src/pages/components/HistoryPage.qml

```diff
@@ -0,0 +1,44 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Browser 1.0
+
+Page {
+    HistoryList {
+        id: view
+
+        anchors.fill: parent
+
+        model: HistoryModel { id: historyModel }
+
+        header: Column {
+            width: parent.width
+            PageHeader {
+                //% "History"
+                title: qsTrId("sailfish_browser-he-history")
+            }
+            SearchField {
+                id: searchField
+                width: parent.width
+                //% "Search"
+                placeholderText: qsTrId("sailfish_browser-ph-search")
+                EnterKey.onClicked: focus = false
+                onTextChanged: {
+                    historyModel.search(searchField.text)
+                    view.search = searchField.text
+                }
+            }
+        }
+        section {
+            property: "sectionName"
+            delegate: SectionHeader { text: section }
```

Comment:
This should be time based on the qml side. On the model you should have something like "dataTimeSection" and for the 
```
SectionHeader {
    text: Format.formatDate(section, Formatter.TimepointSectionRelative)
}
```

Reply:
If I change to data, then each element of the model will be in a own section, so every date will be unique.

Reply:
If I change to data, then each element of the model will be in own section, so each date will be unique